wpeverest/everest-forms
# Fix - Default value missing in form fields

### All Submissions:






### Changes proposed in this Pull Request:

Previously there was an option to select the default values only for some fields. This PR helps to manage the default values for missing fields that need to enhance the option for form default values.


### How to test the changes in this Pull Request:

1. Open **Form Builder** area on Everest Forms.
2. Choose the required field where you want to set the **Default Value**.
3. On the left panel, you will see **Advanced Options**. Click on it and the **Default Value** option will appear.
4. Type any text for default value or select from the **Smart Tag** option which will be later shown in the form for front users.
5. That's all. You can save changes and see the updates in the preview field or form builder inputs.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Field Default Values has been updated for missing fields
